cmd/skywire-visor/commands/root.go

```diff
@@ -275,9 +341,13 @@ func initConfig(mLog *logging.MasterLogger, args []string, confPath string) *vis
 	return conf
 }
 
+// runBrowser is actually not recommended because browser shouldn't run as root
+// but the vpn Client and Server need root to function.
```

Comment:
This should not be in here. Its a consideration we should have as a ticket.

Reply:
removing with next commit